src/main/scala/tech/cryptonomic/nautilus/cloud/adapters/doobie/MeteringStatsQueries.scala

```diff
@@ -0,0 +1,26 @@
+package tech.cryptonomic.nautilus.cloud.adapters.doobie
+
+import doobie.implicits._
+import doobie.util.query.Query0
+import doobie.util.update.Update0
+import tech.cryptonomic.nautilus.cloud.domain.metering.stats.AggregatedMeteringStats
+import tech.cryptonomic.nautilus.cloud.domain.user.User.UserId
+
+/** Queries for metering_stats table */
+trait MeteringStatsQueries {
+
+  /** Gathers last recorded interval per user ID */
```

Comment:
Unclear to me: is this what it does? I'm not sure if that's to say that each user id will have a single result row with its latest interval, or if it's about having all user-grouped intervals, with latest coming out first?
The docs are not that clear to me

Reply:
Each user ID will have single result row with its latest interval. It’s pretty clear to me, but I'm open to the suggestions how to improve it.

Reply:
you mean that on db each user will only have a single entry for the
* environment
* period

like, each user will have a single period of recording session?

Reply:
yes, for now each user will have single period of recording session. In the future it may change, but for now it's like that.

Reply:
thanks for explaining me